api/samples/bbcount.c

```diff
@@ -51,7 +51,7 @@
 #    define DISPLAY_STRING(msg) dr_printf("%s\n", msg);
```

Comment:
In the description:
> Fixes: #4160

I don't know that this will auto-close: the Github docs say to not have a colon and that is what our guide says and all our commits.  (We have a colon on `Issue: #xxx, #yyy` which is for our own parsing.)

Reply:
Do you want to give it a try? or shall we stick to always having no colon for Fixes?

Reply:
https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

Hmm it actually does imply a colon will work in its example.  I wonder if that's new.  I'd be fine with switching to having a colon (do you want to update the wiki?) since it matches typical tags used in commits like `Co-authored-by:` or `Signed-off-by:` and our own `Issue:`.

Reply:
Let's see if it works :p

Reply:
KK, seemed to do the trick. I'll mod the wiki soon.

Reply:
Done.